### PR TITLE
GH-969 Clipboard Changed

### PR DESCRIPTION
**Pull request by Mrnikbobjeff**
### Description of Change ###
Added Clipboard Change Event
- Related to issue #696

### API Changes ###
Added: 
 On Clipboard
- ` event ClipboardChangedEvent<EventArgs>`

### PR Checklist ###

- [ ] Has tests (if omitted, state reason in description)
- [x] Has samples (if omitted, state reason in description)
- [x] Rebased on top of master at time of PR
- [x] Changes adhere to coding standard
- [x] Updated documentation ([see walkthrough](https://github.com/xamarin/Essentials/wiki/Documenting-your-code-with-mdoc))

### CODE REVIEW

**Comment by Mrnikbobjeff** [comment]
Sorry for the delay, had uni stuff coming up. This is basically finished and ready to be merged. I still need to update the docs tomorrow. Tested on Android&UWP, ios is so simple that I dont think i made a mistake there.

File: Samples/Samples/ViewModel/ClipboardViewModel.cs [hunk 1]
```diff
@@ -24,6 +25,21 @@ public string FieldValue
...
+        }
+
+        public override void OnDisappearing()
+        {
+            Clipboard.ClipboardContentChanged -= OnClipboardContentChanged;
+        }
+
+        async void OnClipboardContentChanged(object sender, EventArgs args)
+        {
+            await DisplayAlertAsync("Clipoard content changed!");
```
Comment (mattleibow) [inline Samples/Samples/ViewModel/ClipboardViewModel.cs]: Maybe not have an alert... A status label might be better - you could have a "Clipboard changed: {the text}" or something.

I wonder if there are native features to clear the clipboard for some reason? That might also be a nice feature. But, that can be a separate PR.

Comment (Mrnikbobjeff) [reply Samples/Samples/ViewModel/ClipboardViewModel.cs]: Not all implementations include the changed content, and as we only support text clipboard content I would advise against including the changed content in the event. You can always grab clipboard content in the event handler, but this way we have the bare minimum api across all platforms without fetching the clipboard :)

Comment (mattleibow) [reply Samples/Samples/ViewModel/ClipboardViewModel.cs]: On, no. I was talking about instead of using `DisplayAlertAsync`, rather update some label in the sample UI. I want to move away from popups as they break UI tests and are not as nice as a status label.

Comment (Mrnikbobjeff) [reply Samples/Samples/ViewModel/ClipboardViewModel.cs]: Done

**Comment by Mrnikbobjeff** [comment]
Docs are live, everything should be done. Android device tests should work if you restart them, they did not fault because of me.
